homework/shelkovich_ilya/product/src/main/java/com/tutrit/java/quickstart/service/ProductService.java

```diff
@@ -0,0 +1,19 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Product;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProductService {
+
+    Logger log = LoggerFactory.getLogger("ProductService");
+
+    public double actualPrice(double regularPrice, double discount) {
+        return regularPrice - (regularPrice * discount);
+    }
+
+    public void printInformation(Product product, double actualPrice) {
+        log.info("Product name: {}, regular price: {}, discount: {}%, actual price: {}",
+                product.getName(), product.getRegularPrice(), product.getDiscount() * 100, actualPrice);
+    }
```

Comment:
Log is not a good place to put calculations. Please, add method for that.

Reply:
fixed it